installer/resources/pacbot_app/files/DB.sql

```diff
@@ -3354,5 +3354,8 @@ UPDATE cf_Target SET displayName='PostgreSQL' WHERE targetName in ('cloudsql_pos
 INSERT IGNORE INTO `cf_Target` (`targetName`, `targetDesc`, `category`, `dataSourceName`, `targetConfig`, `status`, `userId`, `endpoint`, `createdDate`, `modifiedDate`, `domain`,displayName)
 VALUES('workstation','Workstation','Compute','crowdstrike','{\"key\":\"_resourceid,cid\",\"id\":\"_resourceid\",\"name\":\"_resourcename\"}','enabled','admin@paladincloud.io',
 concat(@eshost,':',@esport,'/crowdstrike_workstation'),now(),null,'Infra & Platforms','Workstation');
+INSERT IGNORE INTO cf_Target (`targetName`,`targetDesc`,`displayName`,`category`,`dataSourceName`,`targetConfig`,`status`,`userId`,`endpoint`,`createdDate`,`modifiedDate`,`domain`)
+VALUES ('server','Server','Server','Compute','crowdstrike','{"key":"externalAccountId,externalId","id":"externalId","name":"server"}','finding','admin@paladincloud.io','http://vpc-saasdev-data-52k3snqnk3yi63pyocxvhoysyi.us-east-1.es.amazonaws.com:80/crowdstrike_server',now(),null,'Infra & Platforms');
```

Comment:
this is not need to be hardcoded: http://vpc-saasdev-data-52k3snqnk3yi63pyocxvhoysyi.us-east-1.es.amazonaws.com:80/crowdstrike_server

Reply:
good catch, totally missed it. Thanks!